frontend/projects/upgrade/src/app/features/dashboard/feature-flags/pages/feature-flag-root-page/feature-flag-root-section-card/feature-flag-root-section-card.component.html

```diff
@@ -0,0 +1,14 @@
+<app-common-section-card>
+  <app-common-section-card-search-header 
+    header-left
+    [filterOptions]="['All', 'Name', 'Status', 'Context']"
+    [initialFilterOption]="'All'"
+    [initialSearchValue]=""
+    (search)="onSearch($event)"
+  ></app-common-section-card-search-header>
+  <div header-right>right</div>
```

Comment:
both header components should be in one NG container, right?

Reply:
like with the section-list, we chatted in slack and decided that an extra layer here wasn't necessary. That kind of change can be made pretty easily in the future to pull out code into it's own component later  if we find ourselves needing a thing like a second type of header that doesn't work with a left-right layout.